analysis/line-of-sight-geoelement/README.md

```diff
@@ -1,29 +1,35 @@
-# Line of Sight GeoElement
+# Line of sight geoelement
 
 Show a line of sight between two moving objects.
 
-To determine if an observer can see a target, you can show a line of sight between them. The line will be green until it is obstructed, in which case it will turn red. By using the GeoElement variant of the line of sight, the line will automatically update when either GeoElement moves.
+![Image of line of sight geoelement](LineOfSightGeo.jpg)
```

Comment:
the image paths in the common design readmes rarely match the names of those in our existing implementations we noticed: we decided to keep the name of our existing image so as not to add more time going through and renaming images. 
```suggestion
![Image of line of sight geoelement](LineOfSightGeoElement.gif)
```